### PR TITLE
Bug Fixed : Default filters on patients page loads

### LINKED ISSUE #6101: Patients List: Empty emergency phone number filter applied by default on mount
**Describe the bug**
An empty filter for emergency phone numbers (and hence causes `page` and `limit` filters) to be applied too by default.

**To Reproduce**

1. Open https://care.ohc.network
2. Go to patients tab
3. Observe url, it goes to `/patients` and then jumps to `/patients?page=1&limit=12&emergency_phone_number=` immediately

**Expected behavior**

Behavior should be similar like other tab pages (`/assets`, `/facilities`).
The empty `emergency_phone_number` filter shouldn't be applied and hence should be navigated to `/patients` itself by default.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by mustafaazad03**
### WHAT
Solved the bug for An empty filter for emergency phone numbers to be applied by default.

## Proposed Changes
image
image

- Fixes #6101 
- Implement the check for phone number and also make it null
- Implement the check for emergency phone number and make it null
- Make the limit equal to null as a default

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Nice work on fixing it! However, is setting limit=null necessary?

`useFilters`'s limit prop is for identifying the pagination limit when paginating, and is not a default value!

And hence causes to:
image

Rest all LGTM!

**Comment by rithviknishad** [comment]
Btw, you can keep pushing to this current branch instead of always re-opening a new PR! The commits will be squashed and merged anyways once tested!

**Comment by mustafaazad03** [comment]
Ok I will see thanks for pointing out 🙂.

File: src/Components/Patient/ManagePatients.tsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ export const PatientManager = () => {
     FilterBadges,
     resultsPerPage,
   } = useFilters({
-    limit: 12,
+    limit: null,
```
Comment (rithviknishad) [inline src/Components/Patient/ManagePatients.tsx]:
```suggestion
    limit: 12,
```

File: src/Components/Patient/ManagePatients.tsx [hunk 2]
```diff
@@ -121,7 +121,8 @@ export const PatientManager = () => {
 
     if (phone_number === "+91" || phone_number === "") {
       setPhoneNumberError("");
-      updateQuery({ phone_number: "" });
+      qParams.phone_number && updateQuery({ phone_number: null });
+      // updateQuery({ phone_number: "" });
```
Comment (rithviknishad) [inline src/Components/Patient/ManagePatients.tsx]:
```suggestion
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by mustafaazad03** [comment]
Hi @rithviknishad, I update all suggested changes will you please check

**Review by rithviknishad — APPROVED** [review]
Nice work @mustafaazad03! LGTM.

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**